lib/howl/ui/editor.moon

```diff
@@ -647,6 +647,21 @@ class Editor extends PropertyObject
     bar\remove id
     @indicator[id] = nil
 
+  _apply_to_line_modes: (method) =>
+    lines = @active_lines
+
+    mode = nil
+    modes = [@buffer\mode_at line.start_pos for line in *lines]
+    mode = modes[1]
+    for other_mode in *modes
+      if mode != other_mode
+        mode = @buffer.mode
+        break
+
+    @buffer\as_one_undo ->
+      for line in *lines
```

Comment:
Yes, this probably is the source of the linting problem (but I also think that switching to another branch where the config ends up missing might also cause problems). However, the code is wrong anyway - it shouldn't loop over the lines but instead do the op just once, and the `as_one_undo` is not needed.